### PR TITLE
ci: exclude environment from image

### PR DESCRIPTION
**Pull request by JSPRH**
# Description

Since we would rather not have the environment in the images (because they are public), this PR excludes the environment from the image build.

Resulting `app` folder contents for the image:
```bash
/app # ls -la
total 444
drwxr-xr-x    1 root     root          4096 Aug 10 14:06 .
drwxr-xr-x    1 root     root          4096 Aug 10 14:06 ..
-rw-r--r--    1 root     root         34294 Aug 10 13:52 3rdpartylicenses.txt
drwxr-xr-x    2 root     root          4096 Aug 10 13:52 assets
-rw-r--r--    1 root     root         37241 Aug 10 13:52 main.js
-rw-r--r--    1 root     root         93540 Aug 10 13:52 main.js.map
drwxr-xr-x  338 root     root         12288 Aug 10 10:44 node_modules
-rw-r--r--    1 root     root        250839 Aug 10 13:52 package-lock.json
-rw-r--r--    1 root     root          1296 Aug 10 13:52 package.json
```

# Checklist:

- [x] The title of this PR and the commit history is conform with
	the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
- [x] I have performed a self-review of my own code.
- [x] ~~My changes generate no new warnings, SonarCloud reports no Vulnerabilities, Bugs or Code Smells.~~
- [x] ~~I have added tests (unit and E2E if user-facing) that prove my fix is effective or that my feature works,
	Coverage > 80% and not less than the current coverage of the main branch.~~
- [x] The PR branch is up-to-date with the base branch. In case you merged `main` into your feature branch, make sure you have run the latest NX migrations (`nx migrate --run-migrations`).

### CODE REVIEW

File: apps/api/webpack.config.js [hunk 1]
```diff
@@ -1,12 +1,5 @@
 const { composePlugins, withNx } = require('@nx/webpack');
-const CopyPlugin = require('copy-webpack-plugin');
```
Comment (timonmasberg) [inline apps/api/webpack.config.js:2]: can you run npm remove on this package please

Comment (JSPRH) [reply apps/api/webpack.config.js:2]: Done. Thanks for the catch!

**Pull request merged**